language/parser.py

```diff
@@ -0,0 +1,508 @@
+from langkit.parsers import Grammar, List, NoBacktrack as cut, Opt, Or, Pick  # type: ignore
```

Comment:
Is there a specific reason to rename `NoBacktrack`? In most instances where `cut` is used it wouldn't make a large difference.

Reply:
Changed in b5f4b59.